moodler/config.py

```diff
@@ -12,3 +12,6 @@
 
 # List of names of students to not grade
 STUDENTS_TO_IGNORE = env.dict("STUDENTS_TO_IGNORE", subcast_key=int, default={})
+
+LTI_TOKEN = env("LTI_TOKEN")
+LTI_URL = env("LTI_URL")
```

Comment:
is it required or optional? I think it has to be optional as not all courses has LTI integration.

Reply:
Configured to be required but can be left empty.